target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -9,123 +9,110 @@ ath79_setup_interfaces()
 	local board="$1"
 
 	case "$board" in
-	"etactica,eg200")
+	etactica,eg200)
```

Comment:
Please move this, to keep the file consistent.

Reply:
Updated.

Reply:
Sorry I put it at wrong place again...